modules/hcp-get-ocp-channel.adoc

```diff
@@ -132,4 +113,89 @@ version:
         dated 20240826 or newer before updating OCP.
       name: SRIOVFailedToConfigureVF
       url: https://issues.redhat.com/browse/NHE-1171
+# ...
+----
+
+. Update the `spec.release` field of the `HostedCluster` to trigger the control plane rollout by running the following command:
++
+[source,terminal]
+----
+$ oc patch -n <hosted_cluster_namespace> hostedcluster <hosted_cluster_name> --type=merge -p '{"spec":{"release":{"image":"quay.io/openshift-release-dev/ocp-release@sha256:<release_payload>"}}}' <1>
+----
+<1> Replace `<release_payload>` with the {product-title} release payload that you want to use.
+
+. Set your desired channel in your hosted cluster by using one of the following ways:
+
+.. Modify the `HostedCluster` CR by running the following command:
```

Comment:
nit: do we say "CR" in docs?  I'd have expected "resource" here, to avoid the acronym (and hopefully make this more accessible for new readers) and drop the C/custom (because "is this resource custom?" doesn't seem relevant to this procedure).